README.md

```diff
@@ -1,6 +1,6 @@
 # zero-functional
 
-[![Build Status](https://travis-ci.org/alehander42/zero-functional.svg?branch=master)](https://travis-ci.org/alehander42/zero-functional)
+[![Build Status](https://travis-ci.org/alehander42/zero-functional.svg?branch=rework-iterator)](https://travis-ci.org/alehander42/zero-functional)
```

Comment:
I think that was supposed to be reverted before commit

Reply:
did that :)